drv/lpc55-swd/src/main.rs

```diff
@@ -747,7 +994,6 @@ impl ServerImpl {
     }
 
     fn swd_read(&mut self, port: Port, reg: RawSwdReg) -> Result<u32, Ack> {
-        ringbuf_entry!(Trace::SwdRead(port, reg));
```

Comment:
Is removing this logging intentional (also below in `swd_write`)?

Reply:
Yes. In the case of code injection, this is a very noisy point to log.